vm/src/stdlib/os.rs

```diff
@@ -682,6 +682,11 @@ mod _os {
             .to_owned())
     }
 
+    #[pyfunction]
+    fn getcwdb(vm: &VirtualMachine) -> PyResult<Vec<u8>> {
+        Ok(getcwd(vm)?.as_bytes().to_vec())
```

Comment:
to avoid redundant copy, `into_bytes()` is available in `String`

https://doc.rust-lang.org/std/string/struct.String.html#method.into_bytes

Reply:
OK, i fixed it.